packages/components/src/components/telekom/telekom-footer/telekom-footer-content.tsx

```diff
@@ -32,6 +39,8 @@ export class TelekomFooterContent {
                     'var(--telekom-line-weight-highlight) solid var(--telekom-color-functional-focus-on-dark-background)',
                 }}
                 transparent
+                href={this.logoHref}
+                title={this.logoHideTitle ? '' : this.logoTitle}
```

Comment:
I think I would this `undefined` instead of an empty string, because undefined (if I'm not mistaken) won't render the attribute, a empty string could render `title=""` (but I'm not sure!)